test/PositionService.Data.UnitTests/PositionUserRepositoryTests.cs

```diff
@@ -107,16 +107,10 @@ public async Task ShouldAddPositionUserAsync()
         CreatedBy = Guid.NewGuid()
       };
 
-      SerializerAssert.AreEqual(position.Id, await _repository.CreateAsync(position));
+      Assert.DoesNotThrowAsync(async () => await _repository.CreateAsync(position));

```

Comment:
May be replace 'async  Task' on 'void' in the method, like in ShouldReturnNullForAddNullPositionAsync?